lib/jira/verify-installation.js

```diff
@@ -0,0 +1,28 @@
+const getAxiosInstance = require('../jira/client/axios')
+const { Sentry } = require('../error-handler')
+
+module.exports = function (installation, log) {
+  return async () => {
```

Comment:
I am not sure why we are returning a function here? 
Would be great if you could share your thoughts about this.

Reply:
Yeah, it's a bit weird.

We use it in two places. [In the first](https://github.com/integrations/jira/pull/230/files#diff-766fc9b709027b4794f61f1326ab6a21R205), we immediately call the returned function so the outer function serves no purpose. [In the second](https://github.com/integrations/jira/pull/230/files#diff-a7e9a0e95ff84e5221ba1ddbd31f60ddR9), we need to return a callback function that has `installation` and `log` available to it. Because it is called _later_, I've used a function that returns a function to ensure `installation` and `log` within the functions closure. 

I hope that made sense. It was difficult to describe with text. If you know a way around this, I'd happily give you sparkles! ✨

Reply:
Makes sense 👍